File: GEOMETRIA/2840/main.js
function ballons(radius, liters) {
    const pi = 3.1415;
    let volume = (4 / 3) * pi * Math.pow(r, 3);
    return parseInt(liters / volume);
}

let input = require('fs').readFileSync('/dev/stdin', 'utf-8');
let lines = input.split('\n');
let [r, l] = lines[0].split(' ').map(Number);
console.log(ballons(r, l));

File: GEOMETRIA/2840/main.c
#include <math.h>
#include <stdio.h>

int ballons(int radius, int liters) {
    double pi = 3.1415;
    double volume = (4.0 / 3) * pi * pow(radius, 3);
    return (int)(liters / volume);
}

int main() {
    int r, l;
    scanf("%d %d", &r, &l);
    printf("%d\n", ballons(r, l));
    return 0;
}
